notebooks/ImvizExample.ipynb

```diff
@@ -0,0 +1,228 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "handed-person",
+   "metadata": {},
+   "source": [
+    "# Proof of concept of ImViz requirements using glupyter/bqplot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "frozen-oracle",
+   "metadata": {},
+   "source": [
+    "We start off by silencing warnings that can happen when loading data as well as deprecation warnings, for clarity:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "handed-chicago",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.simplefilter('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "advance-exhibition",
+   "metadata": {},
+   "source": [
+    "We start off by looking at some of the basic features using a single-image viewer jdaviz app:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "covered-bikini",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "from jdaviz import ImViz\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "imviz = ImViz()\n",
+    "imviz.load_data('jw01072001001_01101_00001_nrcb1_cal.fits')\n",
```

Comment:
Can we replace this with something anyone have access to? Like something in `astropy-data` or STScI Box URL thingy?

Reply:
Done, for now just using data from data.astropy.org and later we could switch to JWST simulated (or actual eventually!) data